test/resources/model/api/conversation-metadata.json

```diff
@@ -2,8 +2,8 @@
   "client": "cdcm",
   "conversationId": "D-80542-20201120",
   "subject": "MRN: 19GB4S24GC3PPFGVR7",
-  "issueDate": "2020-11-10T15:00:18.000+0000",
-  "senderName": "CDS Exports Team",
+  "issueDate": "2020-11-10T16:00:00.000+0000",
+  "senderName": "Ben Dover",
```

Comment:
This name doesn't seem appropriate for test data in a public HMRC GitHub repository.